feature/mpls/sr/otg_tests/srte_egress/egress_node_forwarding /README.md

```diff
@@ -0,0 +1,190 @@
+# SR-1.2: Egress Node Forwarding
+
+## Summary
+
+This is to test the forwarding functionality of MPLS traffic on an Egress node 
+in SRTE+MPLS enabled network.
+
+The tests validate that the DUT performs the following actions -
+
+ - DUT is an egress node in SRTE+MPLS network.
+ - DUT will receive unlabled traffic for IPv4 destination and perform IPv4 forwarding.
```

Comment:
Is this saying that we're trying to test vanilla IPv4 forwarding with no SR-MPLS involvement? If so, I'm not sure we need a test here for this.

Reply:
This test does not currently exercise PHP in its currrent state. Can we make this explicit that are we are expecting the node to do PHP if that is what we are testing?

Reply:
So for the case where the _previous_ node did PHP, what are we actually testing here? This seems like we're just testing IPv4 forwarding -- do we need a new test for that?

(I'd expect that we need a test for the actual PHP behaviour being implemented at some point -- but this might not be relevant here.)